sass/_pages.scss

```diff
@@ -0,0 +1,16 @@
+.singlePost{
+  .thumbnail{
+    img{
+      width: 100%;
+      height: auto;
+    }
+  }
+  .givery_mission{
```

Comment:
You are using double underscore style in css definition.
So you should not use such a snake case definition.